.github/workflows/ci-pkg-install.yml

```
@@ -63,6 +63,8 @@ jobs:
         run: python -m lightning --version
 
       - name: DocTest package
+        env:
+          PY_IGNORE_IMPORTMISMATCH: 1  # needed for Python 3.7
```

@carmocca @Borda I had to add this here. I couldn't find another solution. The errors I was getting:

https://github.com/Lightning-AI/lightning/actions/runs/3421692912/jobs/5698175906

```
==================================== ERRORS ====================================
_____________ ERROR collecting src/lightning/pytorch/__about__.py ______________
/opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/_pytest/runner.py:339: in from_call
    result: Optional[TResult] = func()
/opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/_pytest/runner.py:370: in <lambda>
    call = CallInfo.from_call(lambda: list(collector.collect()), "collect")
/opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/_pytest/doctest.py:551: in collect
    mode=self.config.getoption("importmode"),
/opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/_pytest/pathlib.py:556: in import_path
    raise ImportPathMismatchError(module_name, module_file, path)
E   _pytest.pathlib.ImportPathMismatchError: ('lightning.pytorch.__about__', '/opt/hostedtoolcache/Python/3.7.15/x64/lib/python3.7/site-packages/lightning/pytorch/__about__.py', PosixPath('/home/runner/work/lightning/lightning/src/lightning/pytorch/__about__.py'))


...
```

Is it ok with you?

---

I don't know what the error is trying to say, but if you want to ignore it, then I'd suggest

```suggestion
        env:
          PY_IGNORE_IMPORTMISMATCH: ${{ matrix.python-version == "3.7" }}
```

---

Sounds good!